src/frontend/src/lib/components/ui/ButtonAuthenticate.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+	import IconArrowAstronaut from '$lib/components/icons/IconArrowAstronaut.svelte';
+</script>
+
+<button
+	on:click
+	class="h-16 px-8 py-2 rounded-full gap-4 font-bold text-lg"
+	data-tid="login-button"
+>
+	<slot />
+	<IconArrowAstronaut />
+</button>
+
+<style lang="scss">
+	button {
+		background: #016dfc;
```

Comment:
Is that an existing color `016dfc`, can we use a variable?
If not, should this color be standadized?

Reply:
i am including it in our variables